### PR TITLE
Consultation Features: Edit (AddToConsult)

### PR DESCRIPTION
**Pull request by wesho1107**
To implement functionality to add students into existing consultations.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddToConsultCommand.java [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    public static final String COMMAND_WORD = "addtoconsult";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a student to a consultation at INDEX."
+        + "Parameters: INDEX (must be a positive integer)"
+        + "[" + PREFIX_NAME + "NAME]...\n"
+        + "Example: " + COMMAND_WORD + " "
+        + PREFIX_NAME + "John Doe ";
+    public static final String MESSAGE_SUCCESS = "New student(s) added to consult at index %1$s: %1$s";
+    public static final String MESSAGE_DUPLICATE_STUDENT = "The student(s) added are already in the consultation";
+    public static final String MESSAGE_PERSON_NOT_FOUND = "No student match given name(s)";
```
Comment (Respirayson) [inline src/main/java/seedu/address/logic/commands/AddToConsultCommand.java]: No student matching the given name(s)? I guess its just small grammar issues

File: src/test/java/seedu/address/testutil/AddToConsultationDescriptorBuilder.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import seedu.address.model.consultation.Consultation;
+import seedu.address.model.person.Name;
+
+/**
+ * A utility class to help with building AddToConsultationDescriptor objects.
+ */
+public class AddToConsultationDescriptorBuilder {
+    private AddToConsultationDescriptor descriptor;
+
+    public AddToConsultationDescriptorBuilder() {
```
Comment (Respirayson) [inline src/test/java/seedu/address/testutil/AddToConsultationDescriptorBuilder.java:17]: perhaps can write the javadoc for these 2 constructors

**Review by Respirayson — APPROVED** [review]
other than a few nits, looks good

File: src/main/java/seedu/address/logic/commands/AddToConsultCommand.java (hunk 1 above)
Comment (wesho1107) [reply src/main/java/seedu/address/logic/commands/AddToConsultCommand.java]: resolved!

**Pull request merged**